### DISCUSSION TITLE
pre-releases

### DISCUSSION THREAD

**Discussion by cchafe** [jacktrip/jacktrip#599] [General]
I tagged rc.4 an hour ago but it still hasn't completed building the installers, so can't test auto update.
I thought that was automatic, right?

Since there are already more updates to dev, we should agree when rc.5 gets tagged and make sure
we're not skewing version numbers.

I'd suggest rc.5 for tonight (Friday, PDT)

**Comment by mattahorton** [comment]
It's automatic. For some reason it thought it needed to rebuild static qt and that takes forever on windows.

**Reply by dyfer** [reply]
> For some reason it thought it needed to rebuild static qt

Yeah... it seems that cache is not available for tagged builds, IIUC.

**Reply by mattahorton** [reply]
they're up now

**Reply by mattahorton** [reply]
And i've got signed versions coming to replace the mac/windows artifacts.

**Comment by cchafe** [comment]
Great -- give me a heads up when the signed ones are there. Thanks!

**Reply by mattahorton** [reply]
@cchafe they're up for rc.4

**Comment by cchafe** [comment]
[win10] 
launched rc.3 and did not get a prompt for upgrade
then installed rc.4 -- signed. Only install question on win10 is "allow msi to make changes?" which I guess is inevitable.
...tested fine

**Comment by cchafe** [comment]
udp-fix #601 merged to dev
 Vs dark mode #593  is pending review, I've never used a dark mode app, so someone who has...
(we will need a PR that will bump to rc.5)
anything else for rc.5?

**Comment by cchafe** [comment]
plc-bcast #604 bumps release to rc.5 and implements broadcast mode under bufstrategy 3 (what could go wrong?)
rc.5 will get the spell checks

...should it also get 
dark mode styling to update text #602
and
dark mode #593
?

**Comment by cchafe** [comment]
rc.5 tagged and pre-released  -- please add its signed assets when possible
